client/plots/matrix.interactivity.js

```diff
@@ -148,12 +148,11 @@ export function setInteractivity(self) {
 		//when clicking a cell in SV, CNV, mutation panels
 		const geneName = sampleData.term?.type == 'geneVariant' ? sampleData.term.name : null
 
-		self.dom.menubody.selectAll('*').remove()
-		self.dom.menutop.selectAll('*').remove()
-		self.dom.tip.show(event.clientX, event.clientY, false, true)
+		if (!self.dom.clickMenu) self.dom.clickMenu = new Menu({ padding: '0px' })
```

Comment:
all menus should be created upfront in constructor

Reply:
Thanks! modified